Dockerfile

```diff
@@ -1,8 +1,26 @@
 ARG GRAFANA_VERSION
 FROM grafana/grafana:$GRAFANA_VERSION as plugins_fetcher
+
+RUN grafana-cli plugins install agenty-flowcharting-panel
```

Comment:
I suggest to add these under a single directive to avoid too many layers (maximum is 42 btw)